.github/workflows/docker.yml

```diff
@@ -0,0 +1,64 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Docker
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push events for the master branch and scheduled nightly runs
+  push:
+    branches:
+      - master
+  schedule:
+      # Nightly run unit tests to check for implicit dependency issues between pull requests.
+      - cron: '0 0 * * *'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+# NOTE: Github actions does not support anchors apparently. Forgive duplication of job
+#       steps.
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      # Cache local python environment artifacts for the current python version
+      # and poetry lockfile hash.
+      - uses: actions/cache@v2
+        id: env-cache
+        with:
+          # Confirmed that the `.local` directory doesn't exist until the
+          # `pip install --user` is invoked below, so we can be confident that
+          # only our python stuff is being captured in this cache (intentional).
+          path: |
+            ~/.cache/pypoetry/virtualenvs/
+            ~/.local
+          key: python-${{ matrix.python-version }}-${{ hashFiles('poetry.lock') }}
+
+      - name: Source .env file
+        run: |
+          source docker/.env
```

Comment:
We should not have to source the file if we tell `docker-compose` to use this `.env` file. Secondly, I'm also not sure if this `source` has any impact outside of this specific `run` statement in actions space?